src/PhpPact/Standalone/ProviderVerifier/Model/VerifierConfig.php

```diff
@@ -9,70 +9,58 @@
  */
 class VerifierConfig implements VerifierConfigInterface
 {
-    /** @var null|UriInterface */
-    private $providerBaseUrl;
+    private ?UriInterface $providerBaseUrl = null;
 
-    /** @var null|string */
-    private $providerStatesSetupUrl;
+    private ?string $providerStatesSetupUrl = null;
 
-    /** @var string */
-    private $providerName;
+    private ?string $providerName = null;
 
-    /** @var string */
-    private $providerVersion;
+    private ?string $providerVersion = null;
 
-    /** @var null|string */
-    private $providerBranch;
+    private ?string $providerBranch = null;
 
-    /** @var array */
-    private $providerVersionTag = [];
+    /**
+     * @var array<int, string>
+     */
+    private array $providerVersionTag = [];
 
-    /** @var bool */
-    private $publishResults = false;
+    private bool $publishResults = false;
 
-    /** @var null|UriInterface */
-    private $brokerUri;
+    private ?UriInterface $brokerUri = null;
 
-    /** @var null|string */
-    private $brokerToken;
+    private ?string $brokerToken = null;
 
-    /** @var null|string */
-    private $brokerUsername;
+    private ?string $brokerUsername = null;
 
-    /** @var null|string */
-    private $brokerPassword;
+    private ?string $brokerPassword = null;
 
-    /** @var string[] */
-    private $customProviderHeaders;
+    /**
+     * @var array<int, string>
+     */
+    private array $customProviderHeaders = [];
 
-    /** @var bool */
-    private $verbose = false;
+    private bool $verbose = false;
 
-    /** @var null|string */
-    private $logDirectory;
+    private ?string $logDirectory = null;
 
-    /** @var string */
-    private $format;
+    private ?string $format = null;
 
-    /** @var int */
-    private $processTimeout = 60;
+    private int $processTimeout = 60;
 
-    /** @var int */
-    private $processIdleTimeout = 10;
+    private int $processIdleTimeout = 10;
 
-    /** @var bool */
-    private $enablePending = false;
+    private bool $enablePending = false;
 
-    /** @var null|string */
-    private $wipPactSince;
+    private ?string $wipPactSince = null;
 
-    /** @var array */
-    private $consumerVersionTag = [];
+    /**
+     * @var array<int, string>
+     */
+    private array $consumerVersionTag = [];
 
-    /** @var ConsumerVersionSelectors */
-    private $consumerVersionSelectors;
+    private ConsumerVersionSelectors $consumerVersionSelectors;
 
-    /** @var null|callable */
+    /** @var ?callable */
```

Comment:
I'm not sure this is valid. I think original code is better:

```suggestion
    /** @var null|callable */
```